TestRunner.psm1

```diff
@@ -154,8 +159,28 @@ function Reset-OdsDatabase {
 
         [Parameter(Mandatory=$True)]
         [string]
-        $DatabaseName
+        $DatabaseName,
+
+        [boolean]
+        $WriteDebugMessages
     )
+
+
+    function Write-DebugLog {
```

Comment:
Looking at the code review, this appears to be a duplicate. If you expand out the lines above, you'll see that this function is embedded inside another function - and we pass that other function to a remote server. Thus this copy of `Write-DebugLog` is only for execution on the remote server.